s3fs/core.py

```diff
@@ -599,7 +599,7 @@ def _open(
         path,
         mode="rb",
         block_size=None,
-        acl="private",
+        acl=None,
```

Comment:
I used False, but same idea. I think the key, though, is not to pass the argument at all when not specified, since S3 should pick whatever it thinks as default for that case.

Reply:
Not really the same because `acl` should be a string. `False` is a boolean.

Reply:
None is also not a string.

Reply:
Updating the doc would be fine, although a falsy value is simply discarded. We are talking about my PR now, not this one, right?